source/04-MX/04.ptx

```diff
@@ -21,35 +21,155 @@ in terms of matrix multiplication.
     </introduction>
 <task>
     <p>
-Create a matrix that doubles the third row of <m>A</m>:
+Which of these tweaks of the identity matrix
+yields a matrix that doubles the third row of <m>A</m> when
+left-multiplying? (<m>2R_3\to R_3</m>)
 <me>
  \left[\begin{array}{ccc} \unknown &amp; \unknown &amp; \unknown \\ \unknown &amp; \unknown &amp; \unknown \\ \unknown &amp; \unknown &amp; \unknown \end{array}\right]
  \left[\begin{array}{ccc} 2 &amp; 7 &amp; -1 \\ 0 &amp; 3 &amp; 2 \\ 1 &amp; 1 &amp; -1 \end{array}\right]
 =
  \left[\begin{array}{ccc} 2 &amp; 7 &amp; -1 \\ 0 &amp; 3 &amp; 2 \\ 2 &amp; 2 &amp; -2 \end{array}\right]
 </me>
+    <ol label="A." cols="2">
```

Comment:
Adding some choices here so folks can at least plug and chug out an answer, but still use inquiry to more cleverly guess which of the choices is most appropriate.